packages/sdk/src/types/runtime/appearance.ts

```diff
@@ -4,10 +4,11 @@
  */
 
 import { Actor, Material } from '.';
+import UserGroup from '../userGroup';
 import { ZeroGuid } from '../../constants';
 
 export interface AppearanceLike {
-    enabled: boolean;
+    enabled: boolean | number;
```

Comment:
Will you be adding this to the behavior system too? Would be good to include that in this PR. Let's get clarity on how UserGroups apply to triggers and user input.

Reply:
I was not planning on adding this to behaviors, no. Instead I would add it to the colliders.

---
In reply to: [266632591](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/269#discussion_r266632591) [](ancestors = 266632591)